internal/commands/build/build.go

```diff
@@ -145,13 +151,21 @@ func buildImageUsingBuildx(app *types.App, contextPath string, opt buildOptions,
 			Driver: d,
 		},
 	}
-	pw := progress.NewPrinter(ctx, os.Stderr, opt.progress)
+
+	var out = os.Stdout
```

Comment:
This was `Stderr` before, any reason it changed? Also, shouldn't it be the out (or err) form dhe dockerCli?

Reply:
this is a limitation from BuildX to require an `os.File` here, we can't use `dockerCli.Out` - unclear to me the impact

This indeed was Stderr as a copy/paste from buildX's `Bake`, but doesn't make sense to me to use error stream to report build steps

Reply:
Because if you pipe it you don't want to have this output, basically : err is for humans, out is for piping. We should keep err here I think since this is what it will be when we move the code to buildx.

Reply:
Do you know any other Unix command (ab)using stderr this way ?

The output without build logs isn't a good candidate to pipe to another command, as it is formatted for human consumption as well :
`Sucessfully built xxxx\nSucessfully tagged foo/bar`

If you want to pipe commands, better use either `-q` or `--iidfile`

Also, the docker cli doesn't use Stderr either https://github.com/docker/cli/blob/master/cli/command/image/build.go#L248

Reply:
OK